xl/trax/track.py

```diff
@@ -660,12 +668,12 @@ def get_tag_search(self, tag, format=True, artist_compilations=False,
         """
         extraformat = ""
         if tag == "albumartist":
-            if artist_compilations and self.__tags.get('__compilation'):
-                value = self.__tags.get('albumartist', None)
-                tag = 'albumartist'
-                extraformat += " ! __compilation==__null__"
```

Comment:
It appears to me that the line is unnecessary. Adding the extraformat to the Albumartist tag simply adds ' ! __compilation==__null__' to the end of the albumartist tag. This does not seem to affect sorting or display but seems like it was used as a way to replicate the function of self.__tags.get('__compilation')